dutch/bos/ontvoering/meehelpen/plassen/lopen/lopen.md

```diff
@@ -4,5 +4,5 @@ Wat doe je?
 Je laat je op de grond vallen en doet alsof je dood bent.
 
 Je klimt in een boom.
-
-Je stopt, [draait je om](omdraaien/terugrennen.md) en rent zo hard als je kan op de zware jongens af.
\ No newline at end of file
+Maar komt [vast te zitten](vastzitten/terugrennen.md).
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.